Node/src/models/index.d.ts

```diff
@@ -213,62 +195,264 @@ export interface O365ConnectorCardActionBase {
+ */
+export interface IO365ConnectorCardFact {
+  name: string;
+  value: string;
+}
+
+export interface IIsO365ConnectorCardFact {
+  toFact(): IO365ConnectorCardFact;
+}
+
+/**
+ * @interface
+ * Interface of O365 connector card image
  *
- * @member {string} [themeColor] Theme color for the card
+ * @member {string} [image] URL for the image
  *
- * @member {array} [sections] Set of sections for the current card
+ * @member {string} [title] Alternative text for the image
  *
  */
-export interface O365ConnectorCard {
+export interface IO365ConnectorCardImage {
+  image: string;
   title?: string;
-  text?: string;
-  summary?: string;
-  themeColor?: string;
-  sections?: O365ConnectorCardSection[];
+}
+
+export interface IIsO365ConnectorCardImage {
+  toImage(): IO365ConnectorCardImage;
 }
 
 /**
- * @class
- * Initializes a new instance of the O365ConnectorCardViewAction class.
- * @constructor
- * @member {string} [name] Name of the action
+ * @interface
+ * Base interface of O365 connector card action.
+ *
+ * @member {string} [type] Type of the action
+ *
+ * @member {string} [name] Name of the action that will be used as button title
  *
- * @member {array} [target] Target urls
+ * @member {string} [id] Action Id
  *
  */
-export interface O365ConnectorCardViewAction extends O365ConnectorCardActionBase {
-  name?: string;
-  target?: string[];
+export interface IO365ConnectorCardActionBase {
+  type: string;
+  name: string;
+  id?: string;
+}
+
+export interface IIsO365ConnectorCardActionBase {
+  toAction(): IO365ConnectorCardActionBase;
+}
+
+/**
+ * @interface
+ * Interface of O365 connector card ViewAction action
+ *
+ * @member {array} [target] Target urls, only the first url effective for card button
+ *
+ */
+export interface IO365ConnectorCardViewAction extends IO365ConnectorCardActionBase {
+  target: string[];
+}
+
+/**
+ * @type
+ * Type of literal strings used for OpenUri target (IO365ConnectorCardOpenUriTarget) operating systems (os).
+ *
+ */
+export type O365ConnectorCardOpenUriOS = 'default' | 'iOS' | 'android' | 'windows';
+
+/**
+ * @interface
+ * Interface of O365 connector card OpenUri target
+ *
+ * @member {O365ConnectorCardOpenUriOS} [os] Target operating system
+ *
+ * @member {string} [uri] Target url
+ *
+ */
+export interface IO365ConnectorCardOpenUriTarget {
+  os: O365ConnectorCardOpenUriOS;
+  uri: string;
+}
+
+/**
+ * @interface
+ * Interface of O365 connector card OpenUri action
+ *
+ * @member {O365ConnectorCardOpenUriOS} [os] Target operating system
+ *
+ * @member {string} [uri] Target url
+ *
+ */
+export interface IO365ConnectorCardOpenUri extends IO365ConnectorCardActionBase {
+  targets: IO365ConnectorCardOpenUriTarget[];
+}
+
+/**
+ * @interface
+ * Interface of O365 connector card HttpPOST action
+ * 
+ * @member {string} [body] Content to be posted back to bots via invoke.
+ *
+ */
+export interface IO365ConnectorCardHttpPOST extends IO365ConnectorCardActionBase {
+  body?: string;
```

Comment:
No, I think you're right, this isn't a real post. Are there any additional requirements on "body" other than it be a string? Like is "1" ok, or must it be a JSON-stringified object (like the "value" of an invoke action)?